code/modules/lighting/~lighting_undefs.dm

```diff
@@ -1,9 +1,12 @@
+#ifdef LIGHTING_INSTANT_UPDATES
 #undef LIGHTING_INTERVAL
```

Comment:
You can safely kill instant updating. Setting the process update rate to one tick is a much better method either way.